modules/contentbox/modules/contentbox-ui/views/adminbar/adminbarCSS.cfm

```diff
@@ -0,0 +1,190 @@
+<!-- Adminbar Styles -->
+<cfoutput>
+    <style>
+    ##cb-adminbar {
+        background: ##333;
+        box-shadow: 1px 1px 3px rgba(0, 0, 0, 0.5);
+        color: white;
+        font-size: .875rem;
+        top: 0;
+        left: 0;
+        padding-right: 10px;
+        position: fixed;
+        right: 0;
+        transition: all 1s ease;
+        -moz-transition: all 1s ease;
+        -webkit-transition: all 1s ease;
+        width: 100%;
+        z-index: 9999;
+    }
+    .cb-adminbar__toggle {
+        background: ##333;
+        border-radius: 0px 0px 5px 5px;
+        padding: 8px;
+        position: absolute;
+        left: 0px;
+        top: 0;
+        transition: transform 1s ease;
+        -moz-transition: transform 1s ease;
+        -webkit-transition: transform 1s ease;
+    }
+    .slide_out .cb-adminbar__toggle {
+        box-shadow: 2px 2px 4px rgb(0 0 0 / 20%); 
+        transform: translateY( 100% );
+    }
+    .slide_out .cb-adminbar__toggle .svg-cheveron {
```

Comment:
Shouldn't this be prefixed too to avoid collisions

Reply:
Agreed.  Done.